README.md

```diff
@@ -229,6 +229,7 @@ There are a few basic types of VHS commands:
 - [`Backspace`](#backspace) [`Enter`](#enter) [`Tab`](#tab) [`Space`](#space): special keys
 - [`Ctrl[+Alt][+Shift]+<char>`](#ctrl): press control + key and/or modifier
 - [`Sleep <time>`](#sleep): wait for a certain amount of time
+- [`Playback@<speed>`](#playback): change playback speed for subsequent commands
```

Comment:
Why not use the existing `SET PlaybackSpeed` inline like `SET TypingSpeed`?

Reply:
I didn't check for an existing pull request before adding the same feature, but reusing `SET PlaybackSpeek`.  I don't feel that strongly about it, but making playback speed congruent with typing speed makes sense to me.  There's an alternative implementation in 5b1c9308cf1f71bcc7bdb602356f575935f36822 using this approach.